packages/sdk/src/utils/filterEmpty.ts

```diff
@@ -8,7 +8,9 @@
  * If `obj` is an empty object, return undefined.
  */
 export default function filterEmpty(obj: any) {
-    if (typeof obj !== 'object' || Object.keys(obj).length) {
+    if (typeof obj === 'object' && obj !== null && !Object.keys(obj).length) {
```

Comment:
This should be a less strict check, to catch `undefined`.

Reply:
Undefined is not of type object, so this PR won't have affected that path. It would `return obj`, which in this case is the same thing as `return undefined`.

Reply:
Ah, true.